osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -187,10 +185,17 @@ public Color4 AccentColour
             }
         }
 
-        public void Add(DrawableHitObject<ManiaHitObject, ManiaJudgement> hitObject)
+        public double TimeSpan

```

Comment:
Duration sounds like a bit of a misnomer... My thinking was that it's a span of time that you see on the screen.

Duration or Length to me implies that it takes some amount of time to do something, but this just represents how "much" time you see on the screen.